packages/office-ui-fabric-react/src/components/pickers/BasePicker.test.tsx

```diff
@@ -43,6 +43,10 @@ const basicSuggestionRenderer = (props: ISimple) => {
   return <div> {props.name} </div>;
 };
 
+const getSuggestions = (root: HTMLElement | Document) => {
+  return document.querySelector<HTMLElement>('.ms-Suggestions');
```

Comment:
you should call `getDocument` from utilities with a valid html element. Otherwise you may get the wrong document in child window rendering scenarios.